examples/api/test/material/text_form_field/text_form_field.1_test.dart

```diff
@@ -0,0 +1,59 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:flutter_api_samples/material/text_form_field/text_form_field.1.dart' as example;
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  testWidgets('Pressing space should focus the next field', (WidgetTester tester) async {
+    await tester.pumpWidget(const example.TextFormFieldExampleApp());
+    final Finder textFormField = find.byType(TextFormField);
+
+    expect(textFormField, findsExactly(5));
+
+    final Finder editableText = find.byType(EditableText);
+    expect(editableText, findsExactly(5));
+
+    List<bool> getFocuses() {
+      return editableText.evaluate()
+        .map((Element finderResult) => (finderResult.widget as EditableText).focusNode.hasFocus)
+        .toList();
+    }
```

Comment:
Do you have a better way to get the focuses?

Reply:
Nothing simpler in mind, I think this utility function is readable as-is 👍